galaxy/files/rules/k8s_container_mapper.py

```diff
@@ -89,7 +89,7 @@ def k8s_container_mapper(tool, referrer, k8s_runner_id="k8s"):
         # 3. If no explicit rule mapping was defined, and it's a tool that
         #    requires galaxy_lib, force the default container. Otherwise,
         #    Galaxy's default container resolution will apply.
-        if tool.id in GALAXY_LIB_TOOLS_UNVERSIONED or ("CONVERTER_" in tool.id and "CONVERTER_" == tool.id[:10]):
+        if tool.id in GALAXY_LIB_TOOLS_UNVERSIONED or ("CONVERTER_" in tool.id and "CONVERTER_" == tool.id[:10]) or tool.is_datatype_converter:
```

Comment:
@almahmoud Since the `.is_datatype_converter` property is available, do we still need `CONVERTER in tool.id`?